sharding/vmc.go

```diff
@@ -57,16 +58,23 @@ func joinValidatorSet(c *Client) error {
 
 	// TODO: Check if account is already in validator set. Fetch this From
 	// the VMC contract's validator set
-	txOps, err := c.createTXOps(depositSize)
-	if err != nil {
-		return fmt.Errorf("unable to intiate the deposit transaction: %v", err)
-	}
+	if c.ctx.GlobalBool(utils.ValidatorSetFlag.Name) {
+
+		log.Info(fmt.Sprintf("Joining Validator Set"))
+		txOps, err := c.createTXOps(depositSize)
+		if err != nil {
+			return fmt.Errorf("unable to intiate the deposit transaction: %v", err)
+		}
+
+		tx, err := c.vmc.VMCTransactor.Deposit(txOps)
+		if err != nil {
+			return fmt.Errorf("unable to deposit eth and become a validator: %v", err)
+		}
+		log.Info(fmt.Sprintf("Deposited 100ETH into contract with transaction hash: %s", tx.Hash().String()))
+
+	} else {
+		log.Info(fmt.Sprintf("Not Joining Validator Set"))
```

Comment:
Unnecessary `fmt.Sprintf`. Please format like a sentence, not capitalizing every word